meeting_minutes/2024/2024-10-30.md

```diff
@@ -0,0 +1,68 @@
+![image](https://user-images.githubusercontent.com/1690898/139102180-5c1e2583-14f1-4f58-ab2b-9e3807ed529c.png)
+
+# Common Security Advisory Framework (CSAF) Technical Committee Working Meeting
+
+- Meeting Date: October 30, 2024
+- Time: 17:00 UTC (19:00 CEST, 13:00 EDT, 10:00 PDT)
+
+## Call to Order and Welcome
+
+Meeting called to order @ 17:03 UTC
+
+## Roll call
+
+Inability to register attendees due to OASIS system challenges.
+
+## Participants
+
+| Given Name | Family Name | Affiliation                                                 | Role          |
+|:-----------|:------------|:------------------------------------------------------------|:--------------|
+| Christoph  | Plutte      | Ericsson                                                    | Voting Member |
+| Denny      | Page        | Individual                                                  | Voting Member |
+| Dina       | Truxius     | Federal Office for Information Security (BSI)               | Voting Member |
+| Feng       | Cao         | Oracle                                                      | Voting Member |
+| JD         | Stefaniak   | Dell                                                        | Member        |
+| Justin     | Murphy      | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Voting Member |
+| Michael    | Reeder      | Dell                                                        | Voting Member |
```

Comment:
I didn't record Michael, when I calculated quorum. Did he join later?
However, we are missing Martin who definitely was there.
```suggestion
| Martin      | Prpic          | Red Hat                                                | Voting Member |
| Michael    | Reeder      | Dell                                                        | Voting Member |

Reply:
He was there:

<img width="310" alt="image" src="https://github.com/user-attachments/assets/e4cea929-5d20-4fd4-8306-a0c05c5ec757">